src/mixins/filing-template-mixin.ts

```diff
@@ -420,27 +442,48 @@ export default class FilingTemplateMixin extends Mixins(AmalgamationMixin, DateM
    * Parses a draft continuation in filing into the store. Used when loading a filing.
    * @param draftFiling the filing body to parse
    */
-  parseContinuationInDraft (draftFiling: any): void {
+  parseContinuationInDraft (draftFiling: ContinuationInFilingIF): void {
+    const continuationIn = draftFiling.continuationIn
+
     // save filing id
     this.setFilingId(+draftFiling.header.filingId)
 
     // restore Entity Type
-    this.setEntityType(draftFiling.continuationIn.nameRequest.legalType)
+    this.setEntityType(continuationIn.nameRequest.legalType)
 
     // restore existing business information
-    if (draftFiling.continuationIn.existingBusinessInfo) {
-      this.setContinuationInBusinessInfo(draftFiling.continuationIn.existingBusinessInfo)
+    if (
+      continuationIn.business &&
+      continuationIn.foreignJurisdiction
+    ) {
+      this.setExistingBusinessInfo({
+        affidavitFileKey: continuationIn.foreignJurisdiction.affidavitFileKey,
+        businessIdentifier: continuationIn.business.identifier,
+        businessLegalName: continuationIn.business.legalName,
+        homeJurisdiction: {
+          country: continuationIn.foreignJurisdiction.country,
+          region: continuationIn.foreignJurisdiction.region
+        },
+        identifier: continuationIn.foreignJurisdiction.identifier,
+        incorporationDate: continuationIn.foreignJurisdiction.incorporationDate,
+        isConfirmed: false, // don't restore confirmation checkbox
+        isUlc: continuationIn.isUlc,
+        legalName: continuationIn.foreignJurisdiction.legalName,
+        mode: continuationIn.mode,
+        status: continuationIn.status,
+        taxId: continuationIn.foreignJurisdiction.taxId
+      })
```

Comment:
As above, if manual/MRAS data was entered, are the properties above correctly set to the store?